images/cfw/opt/x1plus/lib/python/x1plus/services/x1plusd/__init__.py

```diff
@@ -18,6 +19,9 @@ async def main():
     settings = SettingsService(router=router)
     ota = OTAService(router=router, settings=settings)
     ssh = SSHService(settings=settings)
+    if settings.get("polar_cloud", False):
```

Comment:
I probably would spell this key as `polar_cloud.enabled`

Reply:
I actually tried that and was getting errors, so I was going to revisit it later. My next two planned PRs are to start talking to the dbus and interface, and start using X1Plus settings service. I can do that in either order. Let me know what you prefer.